src/utils/childProcess.ts

```diff
@@ -141,9 +147,13 @@ export async function spawnFile(
   args?: string[] | SpawnOptionsLog & SpawnOptionsEncoding,
   options: SpawnOptionsLog & SpawnOptionsEncoding = {}
 ): Promise<{ stdout?: string, stderr?: string }> {
+  let finalArgs: string[] = [];
```

Comment:
The changes around `args` here is to convince TypeScript that this thing will always be a `string[]` (and never an object, etc.)